content/api/hub/plugin_spin_message_trigger.md

```diff
@@ -0,0 +1,73 @@
+title = "Spin Message Trigger"
+template = "render_hub_content_body"
+date = "2022-10-15T00:22:56Z"
+content-type = "text/plain"
+tags = ["trigger", "rust"]
+
+[extra]
+author = "Lee-Orr "
+type = "hub_document"
+category = "Plugin"
+language = "Rust"
+created_at = "2022-10-15T00:22:56Z"
+last_updated = "2022-10-15T00:22:56Z"
+spin_version = ">v1.4"
+summary = "A generic messaging trigger for Fermyon Spin"
+url = "https://github.com/lee-orr/spin-message-trigger"
+keywrods = "trigger, message, NATS"
+
+---
+
+
+A generiuc messaging trigger for [Fermyon Spin](https://github.com/spinframework/spin). Note this is an experimental/work-in-progress repo, and is not production ready at this point. Also - if you have suggestions for improvements, feel free to make them in the Issues tab.
```

Comment:
Not urgent but we should probably change occurrences of this to just "Spin"?

Reply:
Good call; will address in follow-up so the change is easier to see.